### ISSUE TITLE
Any way of binding <escape> to close current tab OR if a window is full-screen, minimize it?

### ISSUE THREAD

**Issue by ghost** [Lord-Kamina/keybinder#40]
I have <Esc> bound to closing the current tab. However, and as a Firefox prompt instructs me, I want to be able, also, to use <Esc> to exit full-screen.

**Comment by Lord-Kamina** [comment]
No, the way this was implemented, you cannot assign a single key to do more than one thing.
